tests/ci/ci.py

```diff
@@ -1282,22 +1292,46 @@ def main() -> int:
                     pass
             if Utils.is_killed_with_oom():
                 print("WARNING: OOM while job execution")
-                error = f"Out Of Memory, exit_code {job_report.exit_code}, after {int(job_report.duration)}s"
+                error_description = f"Out Of Memory, exit_code {job_report.exit_code}"
             else:
-                error = f"Unknown, exit_code {job_report.exit_code}, after {int(job_report.duration)}s"
-            CIBuddy().post_error(error, job_name=_get_ext_check_name(args.job_name))
+                error_description = f"Unknown, exit_code {job_report.exit_code}"
+            CIBuddy().post_error(
+                error_description + f" after {int(job_report.duration)}s",
+                job_name=_get_ext_check_name(args.job_name),
+            )
             if CI.is_test_job(args.job_name):
                 gh = GitHub(get_best_robot_token(), per_page=100)
                 commit = get_commit(gh, pr_info.sha)
                 post_commit_status(
                     commit,
                     ERROR,
                     "",
-                    "Error: " + error,
+                    "Error: " + error_description,
                     _get_ext_check_name(args.job_name),
                     pr_info,
                     dump_to_file=True,
                 )
+
+        if not job_report.job_skipped:
+            print("push finish record to ci db")
+            prepared_events = prepare_tests_results_for_clickhouse(
+                pr_info,
+                [
+                    TestResult(
+                        JOB_FINISHED_TEST_NAME,
+                        FAIL if error_description else OK,
+                        raw_logs=error_description or None,
```

Comment:
so for example in case of oom in perf tests it is supposed to save `Error: Out Of Memory, exit_code -1, after 2024s` as a check status? if yes, it is not quite clear to me how, maybe because i don't see how raw_logs maps into `checks` columns

Reply:
raw_logs maps into test_context_raw
<img width="1606" alt="image" src="https://github.com/user-attachments/assets/bd4562dd-5e94-433f-97b3-4df5a876dd96">